ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -568,17 +568,9 @@ Clean Up DSP Page
     END
     Open Data Science Projects Home Page
     ${projects}=    Get All Displayed Projects
-    ${projects_count}=    Get Length    ${projects}
-    WHILE    ${projects_count} > 0    limit=${projects_count}m
-    ...    on_limit_message=Timeout exceeded waiting for all DS projects deletion
-        Delete Data Science Projects From CLI    ${projects}
-        Reload Page
-        ${reloaded}=    Run Keyword And Return Status
-        ...    SeleniumLibrary.Wait Until Page Contains    Data science projects    timeout=30s
-        IF  ${reloaded}==${FALSE}    SeleniumLibrary.Capture Page Screenshot
-        ${projects}=    Get All Displayed Projects
-        ${projects_count}=    Get Length    ${projects}
-    END
+    Delete Data Science Projects From CLI    ${projects}
+    Reload Page
+    Wait For Dashboard Page Title    Data Science Projects    timeout=30s
```

Comment:
It is not required, since we're already waiting in this loop:
https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource#L191-L196

Reply:
okok, just consider that thw wait which is getting removed here happens on frontend. While the loop you're referring to uses backend which is faster than UI

Reply:
Right, this is a KW for setup or teardown, so running faster in CLI is preferred anyway.

Reply:
Let's hope that this won't cause any race-conditions in the sequential tests execution, though :slightly_smiling_face: